docs/contributing/roadmap.rst

```diff
@@ -0,0 +1,111 @@
+.. _roadmap:
+
+####################
+Development Roadmap
+####################
+
+**Authors**: Anemoi Technical Subgroup (ATS)
+
+**Date**: May 2025
+
+This roadmap serves as a guideline for the future development of the Anemoi project. 
+We aim to maintain flexibility and adaptability, ensuring that the system can evolve
+to meet emerging needs while maintaining a high level of performance and usability.
+The collaboration of diverse institutions will continue to be a core strength, ensuring
+the project meets the evolving demands of weather and climate modeling.
+
+**************
+Our Philosophy
+**************
+
+We aim to build a flexible, extensible, and modular ecosystem that addresses the needs
```

Comment:
I sometimes describe Anemoi as Lego, that ability to take well defined pieces (e.g. layers, training approaches) and combine them in either an existing or new formula.
I can't decide if that "vibe" is already captured here.

Reply:
I gave it a shot in 8948604, can you have a look?